app/views/devise/sessions/new.html.erb

```diff
@@ -22,17 +22,15 @@
   </div>
 
   <div class="form-group">
-    <p>
-    <%= f.submit "Sign in", class:"btn btn-lg btn-primary" %>
-    </p>
-    <p> Or quickly... </p>
+    <%= f.submit "Sign in", class:"form-control btn btn-md btn-primary" %>
+ <!--   <p> Or quickly... </p>
```

Comment:
The checkin/checkout feature is necessary.  Could you uncomment it and make sure it uses the new bootstrap?